### ISSUE TITLE
[Bug]: Incorrect normalization on Gradio inference

### ISSUE THREAD

**Issue by ashwinvaidya17** [open-edge-platform/anomalib#962]
### Describe the bug

Both OpenVINO and Troch models give incorrect inference results when using Gradio inference.

### Dataset

MVTec

### Model

PADiM

### Steps to reproduce the behavior

1. Checkout main branch
2. Enable openvino export in config.yaml
3. Train model `python tools/train.py --model padim`
4. Test inference using torch model
```bash
python tools/inference/gradio_inference.py \                                                                                                                                                                                                                                                                      
    --config ./anomalib/models/padim/config.yaml \
    --weights ./results/padim/mvtec/bottle/run/weights/model-v1.ckpt
```
5. Test inference using OpenVINO model
```bash
python tools/inference/gradio_inference.py \                                                                                                                                                                                                                                                                      
    --config ./anomalib/models/padim/config.yaml \
    --weights ./results/padim/mvtec/bottle/run/openvino/model.bin \ 
 --meta_data ./results/padim/mvtec/bottle/run/openvino/meta_data.json
```

### OS information

OS information:
- OS: Ubuntu 20.04
- Python version:3.8.15
- Anomalib version: main branch
- PyTorch version: 1.13.1+cu116
- CUDA/cuDNN version: 11.5
- GPU models and configuration: GeForce RTX 3090


### Expected behavior

Normalization should be correct

### Screenshots

<img width="565" alt="image" src="https://user-images.githubusercontent.com/17232914/225660446-1bcc8a55-7788-440e-8c03-350cbe789928.png">


### Pip/GitHub

GitHub

### What version/branch did you use?

main

### Configuration YAML

```yaml
dataset:
  name: mvtec
  format: mvtec
  path: ./datasets/MVTec
  category: bottle
  task: segmentation
  train_batch_size: 32
  eval_batch_size: 32
  num_workers: 8
  image_size: 256 # dimensions to which images are resized (mandatory)
  center_crop: null # dimensions to which images are center-cropped after resizing (optional)
  normalization: imagenet # data distribution to which the images will be normalized: [none, imagenet]
  transform_config:
    train: null
    eval: null
  test_split_mode: from_dir # options: [from_dir, synthetic]
  test_split_ratio: 0.2 # fraction of train images held out testing (usage depends on test_split_mode)
  val_split_mode: same_as_test # options: [same_as_test, from_test, synthetic]
  val_split_ratio: 0.5 # fraction of train/test images held out for validation (usage depends on val_split_mode)
  tiling:
    apply: false
    tile_size: null
    stride: null
    remove_border_count: 0
    use_random_tiling: False
    random_tile_count: 16

model:
  name: padim
  backbone: resnet18
  pre_trained: true
  layers:
    - layer1
    - layer2
    - layer3
  normalization_method: min_max # options: [none, min_max, cdf]

metrics:
  image:
    - F1Score
    - AUROC
  pixel:
    - F1Score
    - AUROC
  threshold:
    method: adaptive #options: [adaptive, manual]
    manual_image: null
    manual_pixel: null

visualization:
  show_images: False # show images on the screen
  save_images: True # save images to the file system
  log_images: True # log images to the available loggers (if any)
  image_save_path: null # path to which images will be saved
  mode: full # options: ["full", "simple"]

project:
  seed: 42
  path: ./results

logging:
  logger: [] # options: [comet, tensorboard, wandb, csv] or combinations.
  log_graph: false # Logs the model graph to respective logger.

optimization:
  export_mode: openvino #options: onnx, openvino

# PL Trainer Args. Don't add extra parameter here.
trainer:
  enable_checkpointing: true
  default_root_dir: null
  gradient_clip_val: 0
  gradient_clip_algorithm: norm
  num_nodes: 1
  devices: 1
  enable_progress_bar: true
  overfit_batches: 0.0
  track_grad_norm: -1
  check_val_every_n_epoch: 1 # Don't validate before extracting features.
  fast_dev_run: false
  accumulate_grad_batches: 1
  max_epochs: 1
  min_epochs: null
  max_steps: -1
  min_steps: null
  max_time: null
  limit_train_batches: 1.0
  limit_val_batches: 1.0
  limit_test_batches: 1.0
  limit_predict_batches: 1.0
  val_check_interval: 1.0 # Don't validate before extracting features.
  log_every_n_steps: 50
  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
  strategy: null
  sync_batchnorm: false
  precision: 32
  enable_model_summary: true
  num_sanity_val_steps: 0
  profiler: null
  benchmark: false
  deterministic: false
  reload_dataloaders_every_n_epochs: 0
  auto_lr_find: false
  replace_sampler_ddp: true
  detect_anomaly: false
  auto_scale_batch_size: false
  plugins: null
  move_metrics_to_cpu: false
  multiple_trainloader_mode: max_size_cycle
```


### Logs

```shell
NA
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blaz-r** [comment]
Hello, @ashwinvaidya17 I can check this and try to fix the issue.

**Comment by blaz-r** [comment]
@ashwinvaidya17 I can't reproduce this issue using the current main branch. 
From your commands used for inference, I'd say that you may not be using the right version of anomalib as in the latest version of main branch metadata is passed as `metadata`, while in your case it's `meta_data`.

**Comment by ashwinvaidya17** [comment]
My bad 🙃. Updated my branch. Thanks for checking :)

**Comment by blaz-r** [comment]
Good to hear that it was resolved :)
